src/components/templates/Search/utils.ts

```diff
@@ -66,42 +66,72 @@ export function getSearchQuery(
       `(service.attributes.additionalInformation.categories:\"${categories}\")`
     : text || ''
 
+  if (searchTerm.startsWith('did:op:')) {
+    searchTerm = searchTerm.substring(7)
+  }
   // HACK: resolves the case sensitivity related to dataTokenInfo.symbol
   searchTerm = '*' + searchTerm.toUpperCase() + '*'
   searchTerm = addTypeFilterToQuery(searchTerm, serviceType)
-
   return {
     page: Number(page) || 1,
     offset: Number(offset) || 21,
     query: {
-      query_string: {
-        query: `${searchTerm} -isInPurgatory:true`,
-        fields: [
-          'dataTokenInfo.name',
-          'dataTokenInfo.symbol',
-          'service.attributes.main.name',
-          'service.attributes.main.author',
-          'service.attributes.additionalInformation.description'
-        ],
-        default_operator: 'AND'
+      bool: {
+        must: [
+          {
+            bool: {
+              should: [
+                {
+                  query_string: {
+                    query: `${searchTerm}`,
```

Comment:
`searchTerm` differs in `query_string` you need to manually add `OR` between the words . in `match_prase` it's exactly the string the user inputs.